src/Gui/ViewProviderOrigin.h

```diff
@@ -76,6 +76,9 @@ class GuiExport ViewProviderOrigin : public ViewProviderDocumentObject
 
     /// Returns default size. Use this if it is not possible to determine appropriate size by other means
     static double defaultSize();
+
+    // default color for origin (light-blue stored as 0xRRGGBBAA)
+    static const uint32_t defaultColor = (50<<24) + (150<<16) + (250<<8) + 255;
```

Comment:
Is this really the best way to define this parameter?

Reply:
RGBA seems to be the only common ground between the FreeCAD API and the Coin3D API when you try to set a color with a single variable, see:
https://freecad.github.io/SourceDoc/d9/d0b/classApp_1_1PropertyColor.html
https://www.coin3d.org/Coin/html/classSbColor.html